src/transformers/file_utils.py

```diff
@@ -733,7 +732,7 @@ def http_get(url, temp_file, proxies=None, resume_size=0, user_agent: Union[Dict
         total=total,
         initial=resume_size,
         desc="Downloading",
-        disable=bool(logger.getEffectiveLevel() == logging.NOTSET),
+        disable=bool(logger.getEffectiveLevel() == hf_logging.NOTSET),
```

Comment:
This is a bit mouthful in my opinion.
Following @patrickvonplaten suggestion to had
```
hf_logger.set_verbosity_info()
hf_logger.set_verbosity_warning()
hf_logger.set_verbosity_debug()
```
Could we maybe expose utilities like:
```
hf_logger.is_verbosity_at_least_info()
hf_logger.is_verbosity_at_least_warning()
hf_logger.is_verbosity_at_least_debug()
```
?